graphql_client_cli/src/main.rs

```diff
@@ -38,9 +41,21 @@ enum Cli {
     #[structopt(name = "generate")]
     Generate {
         // should be a glob
-        paths: String,
+        /// Path to graphql query file
         #[structopt(parse(from_os_str))]
-        schema: PathBuf,
+        query_path: PathBuf,
+        /// Path to graphql schema file
+        #[structopt(parse(from_os_str))]
+        schema_path: PathBuf,
+        /// Name of struct
+        selected_operation: String,
+        /// Additional derives
```

Comment:
nitpick: we could add a bit more info: "Additional derives that will be added to the generated structs and enums for the response and the variables"